src/lib/CosignerDataLib.sol

```diff
@@ -0,0 +1,96 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+pragma solidity ^0.8.0;
+
+struct CosignerData {
+    // The time at which the DutchOutputs start decaying
+    uint256 decayStartTime;
+    // The time at which price becomes static
+    uint256 decayEndTime;
```

Comment:
any benefit to including these in extraData as well and having them be only uint32s? that may also allow slightly cleaner reading as you could just have `parseInputOverrides(bytes encodedCosignerData) -> CosignerData` or something